tasks/pruner.yml

```diff
@@ -0,0 +1,66 @@
+- name: Copy pruner service account definition
+  template:
+    src: openshift-serviceaccount.yml.j2
+    dest: "{{ osbs_openshift_home }}/{{ inventory_hostname }}-{{ osbs_namespace }}-serviceaccount-{{ item }}.yml"
+  register: yaml_sa
+  with_items:
+  - "{{ osbs_pruner_serviceaccount }}"
+  tags:
+  - oc
+
+- name: Import pruner service account
+  command: >
+    oc create
+    --namespace={{ osbs_namespace }}
+    --filename={{ osbs_openshift_home }}/{{ inventory_hostname }}-{{ osbs_namespace }}-serviceaccount-{{ osbs_pruner_serviceaccount }}.yml
+  register: service_account_import
+  failed_when: service_account_import.rc != 0 and ('already exists' not in service_account_import.stderr)
+  environment: "{{ osbs_environment }}"
+  when: yaml_sa.changed
+  tags:
+  - oc
+
+- name: Copy cluster rolebinding for pruner service account
+  template:
+    src: "openshift-rolebinding.{{ item.yaml_version | default('v2') }}.yml.j2"
+    dest: "{{ osbs_openshift_home }}/{{ inventory_hostname }}-{{ osbs_namespace }}-rolebinding-{{ item.name }}.yml"
+  with_items:
+  - name: build-pruner
+    role: "{{ osbs_pruner_clusterrole_build }}"
+    yaml_version: v1
```

Comment:
Should this be `v2`? the `v1` template doesn't read from `serviceaccounts`.

Maybe remove this line, and hard code `v2` in the `template.src` above?

Reply:
Ah, will change to `v2`.

The template is shared with tasks in main.yml, so I don't think I can change it.

Reply:
I don't mean change the actual template, but change the template key in this task to:
```
src: "openshift-rolebinding.v2.yml.j2"
```

Reply:
Gotcha. Fixed.